beekeeper-core/src/test/java/com/expediagroup/beekeeper/core/repository/HousekeepingMetadataRepositoryTest.java

```diff
@@ -252,12 +253,115 @@ public void findRecordForCleanupByDatabaseAndTableMixedHousekeepingStatus() {
     compare(result.get(), housekeepingTable1);
   }
 
-  private HousekeepingMetadata createEntityHousekeepingTable() {
+  @Test
+  public void countPartitionsForGivenDatabaseAndTableWherePartitionIsNotNull() {
+    HousekeepingMetadata housekeepingTable1 = createPartitionedEntityHousekeepingTable();
+    housekeepingMetadataRepository.save(housekeepingTable1);
+
+    HousekeepingMetadata housekeepingTable2 = createEntityHouseKeepingTable(DATABASE_NAME, TABLE_NAME + "2", null);
+    housekeepingMetadataRepository.save(housekeepingTable2);
+
+    long result = housekeepingMetadataRepository
+        .countRecordsForGivenDatabaseAndTableWherePartitionIsNotNull(DATABASE_NAME, TABLE_NAME);
+
+    assertEquals(Long.valueOf(1), result);
```

Comment:
```suggestion
    assertEquals(1L, result);
```
Can you not just do that? ^